bundles/org.openhab.binding.dbquery/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.dbquery</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: DBQuery Binding</name>
+
+  <properties>
+    <bnd.importpackage>
+      !javax.annotation;!android.*,!com.android.*,!com.google.appengine.*,!dalvik.system,!kotlin.*,!kotlinx.*,!org.conscrypt,!sun.security.ssl,!org.apache.harmony.*,!org.apache.http.*,!rx.*,!org.msgpack.*
+    </bnd.importpackage>
+  </properties>
+
+  <dependencies>
+    <!-- influxdb-client-java -->
+    <dependency>
+      <groupId>com.influxdb</groupId>
```

Comment:
Did you consider adding the dependencies via Karaf (feature.xml), so that the dependencies need to be downloaded only once if the influxdb persistence is also installed? (To make this work, the same adjustment needs to be done to the persistence addon)

Reply:
My intent, if it's possible, is to maintain independence between the persistence plugins.
To be able to evolve differently if needed and use different versions or implementations in the future if it's desired.

Reply:
If the versions differ, both versions will be used. But if the versions are identical, the dependency will be reused. You won't introduce a dependency between the two.

Reply:
You're right.
Ok, I will try then but will need some support and guidance on that.
Is there any recent PR or documentation that I can take as a reference on how to do it?

Also, I will try to do all corrections in a row when you finish the review.

Reply:
You could take a look here:  https://github.com/openhab/openhab-addons/blob/d36e1f10180f42ebe210f9178ab8af2a17a817b6/bundles/org.openhab.binding.ahawastecollection/src/main/feature/feature.xml#L7

Reply:
@fwolter: It's only adding that line and leaving the pom the same way?
Or it's needed to create the bundle too?

Reply:
You don't need to create any bundle. But you need to change the scope in the pom.xml from compile to provided.

Reply:
If it's possible I prefer to leave this as it is for this first version.
Because I prefer to don't put more changes including modification of persistence addon.